rmw_fastrtps_shared_cpp/src/rmw_request.cpp

```diff
@@ -17,8 +17,6 @@
 #include "fastcdr/Cdr.h"
 #include "fastcdr/FastBuffer.h"
 
-#include "fastrtps/subscriber/Subscriber.h"
```

Comment:
I suggest including `fastdds/rtps/common/WriteParams.h` which is being used in this file